### DISCUSSION TITLE
Cypress 10.0.1 Test Runner window // Scroll for list of my test doesn't work. List goes up to the 1st test

### DISCUSSION THREAD

**Discussion by MariaDiamond** [cypress-io/cypress#22063] [Questions and Help]
I migrated to 10.0.1
I started Cypress
I clicked on Chrome to run e2e tests
I see Test Runner separate window with all specs I have under my project
I try to scroll down that list and stay in the middle or at any point down the list of test

Result:
The list automatically goes up, jumps back to the very 1st test in my list.
I can't scroll down.

**Comment by samnittala** [comment]
Have the same problem. There is something wrong with the new Test Runner Window.

**Comment by edaldridge** [comment]
I have seen the same issue, the test runner works ok the first time I use it but if I navigate away from my spec list then go back to it the spec list won't collapse, every time you click the arrow to close a set of test specs it reopens, every time you try to scroll it forces itself back to the top.

I've been able to work around it by using the search function
